.env.develop

```diff
@@ -1,4 +1,5 @@
 # feature flags
+REACT_APP_FEATURE_LIMIT_ORDERS=true
```

Comment:
We'll probably wanna put this in `.env.dev` too, as I think thats the defacto local env.